### PR TITLE
[Doc Improvement][Deep link update for Sharepoint Online]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/16f05cc2c6cddce75e030be4acdaa8b42c3b85c4/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-teams.md) | 90 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e963581c-9f37-4caa-bc6b-0868dc967a52) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/deep-link-teams.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ Use the following deep link format can be used in a bot, connector, or message e
 
 The query parameters are:
 
-* `fileId`: Unique file ID from Sharepoint Online, also known as `sourcedoc`. For example, `1FA202A5-3762-4F10-B550-C04F81F6ACBD`.
+* `fileId`: Unique file ID from Sharepoint Online, also known as `sourcedoc` or `sharepointIds.listItemUniqueId` from Sharepoint Online or `driveItem` API response. For example, `1FA202A5-3762-4F10-B550-C04F81F6ACBD`.
```
Comment (shash-coder) [inline msteams-platform/concepts/build-and-test/deep-link-teams.md:167]: **or** `driveItem` API response
Can you please remove the extra "or" above.

Comment (v-npaladugu) [reply msteams-platform/concepts/build-and-test/deep-link-teams.md]: Updated

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9bc76f87135dd01ab3387377461770c5e387ebd7/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-teams.md) | 90 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a471cad5-0c01-4a3b-b85d-fd052f0d9b7e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by shash-coder — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/deep-link-teams.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d45f038e74f2e19074e035f8c57278a3f99c243b/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fdeep-link-teams.md) | 90 | 14 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3e350200-724a-43d0-9f8f-9d22e511a7f5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**